src/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProvider.php

```diff
@@ -80,6 +98,7 @@ public function getData($id)
             'options' => $this->extractOptionsData($productForEditing),
             'suppliers' => $this->extractSuppliersData($productForEditing),
             'customizations' => $this->extractCustomizationsData($productForEditing),
+            'combinations' => $this->getCombinationsData($productId),
```

Comment:
The combinations MUST not be included in this data provider, one of the purpose of the new page is to split combinations from the rest of the page to improve performance

Even if this is a small subset of combinations and only used for first init it should remain independent and be fetched by the Combinations component via an ajax request